README.md

```diff
@@ -60,32 +70,42 @@ Also check the 'samples' subfolder. It's a maven project, and it contains some c
 
 # Sample code
 
+
 Getting a single frame from a movie ( supports only AVC, H.264 in MP4, ISO BMF, Quicktime container ):
 ```java
-    int frameNumber = 150;
-    BufferedImage frame = AWTFrameGrab8Bit.getFrame(new File("filename.mp4"), frameNumber);
-    ImageIO.write(frame, "png", new File("frame_150.png"));
+int frameNumber = 42;
+Picture picture = FrameGrab.getFrameFromFile(new File("video.mp4"), frameNumber);
+
+//for JDK (jcodec-javase)
+BufferedImage bufferedImage = AWTUtil.toBufferedImage(picture);
+ImageIO.write(bufferedImage, "png", new File("frame42.png"));
+```
+
+Get all frames from a video file
```

Comment:
Could you also add the example for Android? It's basically the same as AWT* bit with Android* and Bitmap class instead of the BufferedImage.